src/blacklist.cpp

```diff
@@ -38,13 +38,14 @@ static bool check_blacklisted() {
         "Steam.exe",
         "ffxivlauncher.exe",
         "ffxivlauncher64.exe",
+        "LeagueClient.exe",
+        "wine-LeagueClientUxRender",
+        "lutris"
```

Comment:
Are you sure? `lutris` is a python script and `proc_name` would be some variation of `python`

Reply:
Yep! Log does show that `python3.8` was blacklisted, but only adding `lutris` allows lutris to launch.

Reply:
Weird, does it show up in the log as "blacklisted"? Could be some utility app that lutris uses like glxinfo etc. Lutris seems to work ok for me either way :confused:

Reply:
Yep, seems that segfault with lutris was not releated, removed from blacklist.